### PR TITLE
solution and error handling task 6

### PR DESCRIPTION
**Pull request by hja7333**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -88,9 +87,50 @@ describe("app", () => {
...
+            votes: 0,
+          },
+          {
+            article_id: 3,
+            author: "icellusedkars",
+            body: "git push origin master",
+            comment_id: 10,
+            created_at: "2020-06-20T07:24:00.000Z",
+            votes: 0,
+          },
```
Comment (pippaamy) [inline __tests__/app.test.js:119]: Needs a 200 test to check that it returns empty array if article has no comments

**Review by pippaamy — CHANGES_REQUESTED** [review]
Another test to add and then pop another pr in :)

**Comment by pippaamy** [comment]
all good merge away!
